app/assets/javascripts/productions.js

```diff
@@ -0,0 +1,82 @@
+$(document).ready(function(){
+	console.log("document ready")
+	toggleMenus();
+    bindListeners();
```

Comment:
Install JSFormat in Sublime to automatically indent and format your code. https://github.com/jdc0589/JsFormat